python/paddle/fluid/layers/nn.py

```diff
@@ -101,9 +101,6 @@
     'pad',
     'roi_pool',
     'roi_align',
-    'image_resize',
```

Comment:
shall we also delete `def image_resize` in near L7500？

Reply:
The function of "def image_resize" will be deleted in later commits.

Reply:
> shall we also delete `def image_resize` in near L7500？

The function of "def image_resize" will be deleted in later commits.

Reply:
Done. Add image_resize back to __all__ list.